libexec/alluxio-config.sh

```diff
@@ -90,33 +90,57 @@ ALLUXIO_SERVER_CLASSPATH="${ALLUXIO_CONF_DIR}/:${ALLUXIO_CLASSPATH}:${ALLUXIO_AS
 ####################################################################################################
 function getConf {
   "${JAVA}" -cp ${ALLUXIO_CLIENT_CLASSPATH} ${ALLUXIO_JAVA_OPTS} -Dalluxio.logger.type=Null \
+      -Dalluxio.remote.logger.type=Null \
```

Comment:
Yes, we need to set the default to Null otherwise log4j will print error messages to stderr, complaining "cannot find value for key log4j.appender.""".